tutorials/algorithms/11_gradients_framework.ipynb

```diff
@@ -0,0 +1,718 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    state_grad_result = gradient.run(qc, H,  params).result().gradients
</pre><p>Instead of <code>state_grad_result</code>, you could name this variable sth like <code>estimator_grad_result</code>, and similarly with the sampler gradient one.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1463/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>